pkg/antctl/raw/multicluster/destroy.go

```diff
@@ -0,0 +1,51 @@
+// Copyright 2022 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package multicluster
+
+import (
+	"strings"
+
+	"github.com/spf13/cobra"
+
+	"antrea.io/antrea/pkg/antctl/raw/multicluster/common"
+)
+
+var destroyOpts *common.CleanOptions
+
+var destroyExamples = strings.Trim(`
+# Destroy ClusterSet in the leader cluster
+  antctl mc destroy --namespace antrea-multicluster --clusterset test-clusterset
+`, "\n")
+
+func NewDestroyCommand() *cobra.Command {
+	command := &cobra.Command{
+		Use:     "destroy",
+		Short:   "Destroy ClusterSet in the given Namespace of the leader cluster",
+		Args:    cobra.MaximumNArgs(0),
+		Example: destroyExamples,
+		RunE:    destroyRunE,
+	}
+
+	o := common.CleanOptions{}
+	destroyOpts = &o
+	command.Flags().StringVarP(&o.Namespace, "namespace", "n", "kube-system", "Antrea Multi-cluster Namespace. Default to kube-system")
```

Comment:
This is the leader cluster Namespace. Cannot be "kube-system". Remove the default value.

Reply:
Maybe you want to add it to "join"?

Reply:
Removed the default Namespace now.